http_blocks

```diff
@@ -1 +1 @@
-Subproject commit b7aa43641182651eec5d6c71a2d31330049b1d97
```

Comment:
Take a look at my comments here: https://github.com/nio-blocks/http_blocks/pull/45

If you agree that that PR should not be merged, then please update this one to not include this submodule update.